operator/pkg/model/ingestion/ingress.go

```diff
@@ -90,6 +90,11 @@ func Ingress(ing slim_networkingv1.Ingress, defaultSecretNamespace, defaultSecre
 		}
 
 		l.Hostname = host
+		if rule.HTTP == nil {
```

Comment:
An additional aspect to this problem is that we actually don't validate (and potentially reject) the CRD.

According to the Ingres spec `HTTP` needs to be set anyway.

```
// IngressRuleValue represents a rule to apply against incoming requests. If the
// rule is satisfied, the request is routed to the specified backend. Currently
// mixing different types of rules in a single Ingress is disallowed, so exactly
// one of the following must be set.
type IngressRuleValue struct {
	// +optional
	HTTP *HTTPIngressRuleValue `json:"http,omitempty" protobuf:"bytes,1,opt,name=http"`
}
```